nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const router = express.Router();
+
+const reviews = require("../data/reviews.json");
+
+router.get("/", async (request, response) => {
+    try {
+        console.log(reviews);
+        console.log("in /api/reviews");
+        response.send(reviews);
+    } catch (error) {
+        throw error;
+    }
+});
+router.get("/:id", async (request, response) => {
+
+    if (typeof parseInt(request.params.id) == "number") {
+        const responseData = reviews.filter(review => review.id === parseInt(request.params.id))
+        if (responseData.length === 0) {
+            return res.status(404).send("Given query does not find any data")
```

Comment:
'Res' is not defined. Should be changed to 'response', since this is what you wrote in line 6.

Reply:
fixed